### ISSUE TITLE
Missing docker-compose.yml

### ISSUE THREAD

**Issue by LegendOfLynkle** [CyberCX-STA/PurpleOps#1]
## Expected Behavior
`docker compose up -d` should build and run the application.

## Actual Behavior
No compose file is present and the command fails with:
`no configuration file provided: not found`

## Steps to Reproduce the Problem
  1. Clone the repository: `git clone https://github.com/CyberCX-STA/PurpleOps`
  2. Go into the repository: `cd PurpleOps`
  3. Run the app with Docker: `docker compose up -d`
  4. Docker compose will fail due to missing configuration file: ![CleanShot 2023-08-15 at 20 21 32@2x](https://github.com/CyberCX-STA/PurpleOps/assets/16172627/7b21b45b-0c30-4b10-a7ae-1c74a2269a82)

**Comment by joda32** [comment]
Hi Lynkle,

Thanks for pointing that out, I was actually working on ripping out the uwsgi component and replacing it with gunicorn. I've just pushed the changes and the docker files are now included.

Just one thing, and I'll update the readme shortly, but running it the first time run it without the -d flag for docker compose. The reason is that the generated admin creds are output on the console to avoid you having to dive into the container to extract it from the .env file.

![image](https://github.com/CyberCX-STA/PurpleOps/assets/5188052/f734bbbf-3da0-44a6-b2f9-8bf837f7def8)

Closing for now
